tests/fixtures/processing/kafka.py

```diff
@@ -88,22 +116,16 @@ def get_expected_dataframe(  # noqa: WPS463
     ) -> pandas.DataFrame:
         pass
 
-    def assert_equal_df(
+    def json_serialize(
```

Comment:
```suggestion
    def json_deserialize(
```
Тут же из json распаковываются значения